reactjs/components/atoms/Paragraph/List/List.js

```diff
@@ -30,15 +30,17 @@ class List extends React.Component {
         Wrapper = 'div';
       }
 
+    const getKey = (index) => `${id}_${index}`;
```

Comment:
Doesn't have to be a function, could be just a const.

Reply:
Could you please provide recommended approach? I'm not sure how else to do it.

Reply:
just `const key = id + '_' + index;`. Then use this var instead of getKey() calls down.

Reply:
That doesn't look right to me. **index** variable is defined in scope of map method below.

Reply:
Good point, I thought it comes from the parent element 👍